vertical-pod-autoscaler/README.md

```diff
@@ -45,13 +45,14 @@ procedure described below.
 
 # Installation
 
-The current default version is Vertical Pod Autoscaler 0.11.0
+The current default version is Vertical Pod Autoscaler 0.12.0
 
 ### Compatibility
 
 | VPA version     | Kubernetes version |
 |-----------------|--------------------|
-| 0.11            | 1.22+              |
```

Comment:
Does this mean that 0.11 will fail when being used with k8s 1.25?

Reply:
Yes. When I start VPA 0.10.0 in a 1.25 cluster the recommender doesn't work properly, in logs I see just  (the last log repeats):

```
$ kubectl logs  vpa-recommender-756fd8fdb9-tmkkb -n kube-system
I0926 10:57:43.903053       1 main.go:74] Vertical Pod Autoscaler 0.10.0 Recommender: default
I0926 10:57:44.005325       1 controller_fetcher.go:142] Initial sync of ReplicationController completed
I0926 10:57:44.105529       1 controller_fetcher.go:142] Initial sync of Job completed
E0926 10:57:44.110317       1 reflector.go:178] k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher.go:137: Failed to list *v1beta1.CronJob: the server could not find the requested resource
E0926 10:57:45.445792       1 reflector.go:178] k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher.go:137: Failed to list *v1beta1.CronJob: the server could not find the requested resource
E0926 10:57:47.748662       1 reflector.go:178] k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher.go:137: Failed to list *v1beta1.CronJob: the server could not find the requested resource
E0926 10:57:52.309820       1 reflector.go:178] k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher.go:137: Failed to list *v1beta1.CronJob: the server could not find the requested resource
```

Reply:
(I get the same behavior for VPA 0.11.0 in a 1.25 cluster)